### ISSUE TITLE
C192mx025 Support

### ISSUE THREAD

**Issue by NeilBarton-NOAA** [NOAA-EMC/global-workflow#2897]
### What new functionality do you need?

Targeted resolution of SFS is C192mx025, hence support is needed for this resolution. Wave grid TBD, but a good default is the GEFS wave grid. 

### What are the requirements for the new functionality?

Scripts that support forecast-only C192mx025. 

### Acceptance Criteria

forecast-only C192mx025 runs in ensemble mode.

**Comment by benjamin-cash** [comment]
Hi @NeilBarton-NOAA - can I assume from this issue that the workflow does not currently support this configuration?

**Comment by KateFriedman-NOAA** [comment]
@NeilBarton-NOAA is this issue still a need, let us know, thanks!

**Comment by NeilBarton-NOAA** [comment]
@KateFriedman-NOAA I do not believe so, but I haven't tested this configuration. I fine with closing and re-opening if needed.

**Comment by KateFriedman-NOAA** [comment]
Sounds good, thanks @NeilBarton-NOAA !
